scylla_api_client/api.py

```diff
@@ -401,3 +402,33 @@ def load(self):
                 command.load_json(command_json)
                 module.add_command(command)
             self.add_module(module)
+        for module_def in [json.loads('{ "path": "/v2", "description": "V2 API"}')]:
```

Comment:
it's a shame that the code that creates the commands is duplicated (i.e. v1 vs v2)

Reply:
I'll see what it takes to move it forward to the function suffix or use a shared helper function